src/brevitas_examples/bnn_pynq/models/resnet.py

```diff
@@ -1,28 +1,34 @@
 # Copyright (C) 2023, Advanced Micro Devices, Inc. All rights reserved.
 # SPDX-License-Identifier: BSD-3-Clause
 
-from typing import List, Optional, Tuple, Union
+from typing import List
 
-import torch
 from torch import Tensor
 import torch.nn as nn
-import torch.nn.functional as F
 
-from brevitas.export import export_qonnx
+from brevitas.inject.defaults import Int8WeightPerTensorFloat
 import brevitas.nn as qnn
 from brevitas.quant import TruncTo8bit
 from brevitas.quant_tensor import QuantTensor
 
 
 def make_quant_conv2d(
-        in_channels, out_channels, kernel_size, weight_bit_width, stride=1, padding=0, bias=False):
+        in_channels,
+        out_channels,
+        kernel_size,
+        weight_bit_width,
+        stride=1,
+        padding=0,
+        bias=False,
+        weight_quant=Int8WeightPerTensorFloat):
```

Comment:
I would remove this default since it's defined in the caller already, and for quantizers I prefer to have defaults set only once.